tokio/src/lib.rs

```diff
@@ -433,4 +433,9 @@ cfg_macros! {
 // TODO: rm
 #[cfg(feature = "io-util")]
 #[cfg(test)]
-fn is_unpin<T: Unpin>() {}
```

Comment:
Why is this changing?

Reply:
I'm temporarily breaking formatting in order to see how the message would look.

I was hoping to create an annotation, but it looks relatively complicated for what this is doing. In your opinion is the current message printed in the error log clear enough / easily findable?